projects/ngneat/reactive-forms/src/lib/formControl.spec.ts

```diff
@@ -14,6 +15,49 @@ const validatorExample = new FormControl<string, NgValidatorsErrors>('', {
   }
 });
 
+describe('FormControl valueChanges$ diff() operator', () => {
+  const control = new FormControl<string>();
```

Comment:
Do we support `FormControl([1,2, 3])` and `FormControl({ a: 1, b: 2 })`?

Reply:
Sorry that it was so much time.

Yes I've extended it with test case.